packages/app-store/routing-forms/trpc/response.handler.ts

```diff
@@ -102,34 +110,19 @@ export const responseHandler = async ({ ctx, input }: ResponseHandlerOptions) =>
       },
     });
 
-    const settings = RoutingFormSettings.parse(form.settings);
     let userWithEmails: string[] = [];
-    if (form.teamId && settings?.sendUpdatesTo?.length) {
-      const userEmails = await prisma.membership.findMany({
```

Comment:
This query is redundant (it already was) as the serializableForm had a members listing; which is now used.